src/dd-resizable.ts

```diff
@@ -132,7 +132,7 @@ export class DDResizable extends DDBaseImplement implements HTMLElementExtendOpt
   protected _mouseOver(e: Event): void {
     // console.log(`${count++} pre-enter ${(this.el as GridItemHTMLElement).gridstackNode._id}`)
     // already over a child, ignore. Ideally we just call e.stopPropagation() but see https://github.com/gridstack/gridstack.js/issues/2018
-    if (DDManager.overResizeElement || DDManager.dragElement) return;
+    if (DDManager.overResizeElement === this || DDManager.dragElement) return;
     DDManager.overResizeElement = this;
```

Comment:
by doing this you are not clearing the other (if any) element style at line 148 this.el.classList.add('ui-resizable-autohide') so now 2 items are showing resize ?
would be relly helpfull to have a running example to test the issue first...

Reply:
will fix myself...